zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -1200,6 +1200,31 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
     return true;
   };
 
+  $scope.userHasWritePermission = function() {
+    let owners = $scope.permissions.owners;
+    let writers = $scope.permissions.writers;
+
+    if (owners.length === 0 || writers.length === 0) {
```

Comment:
owners.length shouldn't be empty??

Reply:
if writers.length == 0 shouldn't this return false?

Reply:
If the field is empty, it means that anyone can take this action.
I understand correctly?